src/content/products/deepsparse/community.mdx

```diff
@@ -42,28 +42,42 @@ index: 1000
   </div>
 </div>
 
-<p><br></p>
+DeepSparse is an inference runtime that offers GPU-class performance on CPUs by utilizing sparsity. DeepSparse accepts models in the ONNX format, giving you flexibility to serve your model in a framework-agnostic manner. 
 DeepSparse Community Edition is open-source and free for evaluation, research, and non-production use with our [Engine Community License](https://neuralmagic.com/legal/engine-license-agreement/). (Alternatively, the [Enterprise Edition](https://docs.neuralmagic.com/products/deepsparse-ent) requires a Trial License or can be fully licensed for production, commercial applications.)
 
-## Hardware Support and System Requirements
+Neural Magic's DeepSparse is able to integrate into popular deep learning libraries (e.g., Hugging Face, Ultralytics) allowing you to leverage DeepSparse for loading and deploying sparse models with ONNX.
+ONNX gives the flexibility to serve your model in a framework-agnostic environment.
+Support includes [PyTorch,](https://pytorch.org/docs/stable/onnx.html) [TensorFlow,](https://github.com/onnx/tensorflow-onnx) [Keras,](https://github.com/onnx/keras-onnx) and [many other frameworks](https://github.com/onnx/onnxmltools).
+
+DeepSparse is available in two editions:
+1. [**DeepSparse Community**](#installation) is open-source and free for evaluation, research, and non-production use with our [Engine Community License](https://neuralmagic.com/legal/engine-license-agreement/).
+2. [**DeepSparse Enterprise**](https://docs.neuralmagic.com/products/deepsparse-ent) requires a Trial License or [can be fully licensed](https://neuralmagic.com/legal/master-software-license-and-service-agreement/) for production, commercial applications.
+
+## Features
+
+- 🔌 [DeepSparse Server](https://github.com/neuralmagic/deepsparse/tree/main/src/deepsparse/server)
+- 📜 [DeepSparse Benchmark](https://github.com/neuralmagic/deepsparse/tree/main/src/deepsparse/benchmark)
+- 👩‍💻 [NLP and Computer Vision Tasks Supported](https://github.com/neuralmagic/deepsparse/tree/main/examples)
+
+## 🧰 Hardware Support and System Requirements
 
 Review [CPU Hardware Support for Various Architectures](https://docs.neuralmagic.com/deepsparse/source/hardware.html) to understand system requirements.
-The DeepSparse Engine works natively on Linux. Mac and Windows require running Linux in a Docker or virtual machine; it will not run natively on those operating systems.
+DeepSparse works natively on Linux. Mac and Windows require running Linux in a Docker or virtual machine; it will not run natively on those operating systems.
 
-The DeepSparse Engine is tested on Python 3.7-3.10, ONNX 1.5.0-1.12.0, and ONNX opset version 11+. It is manylinux compliant.
+DeepSparse is tested on Python 3.7-3.10, ONNX 1.5.0-1.12.0, and ONNX opset version 11+, and manylinux compliant.
```

Comment:
```suggestion
DeepSparse is tested on Python 3.7-3.10, ONNX 1.5.0-1.12.0, and ONNX opset version 11+. It is manylinux compliant.
```

Reply:
The reason Beth made this edit is because you wouldn't say "tested on manylinux compliant" which is how the sentence read. If we were to keep it as is, it would need to read 
", and manylinux compliant systems." or what she offered for example.